x-pack/plugins/ml/public/file_datavisualizer/components/about_panel/about_panel.js

```diff
@@ -73,6 +74,14 @@ export function LoadingPanel() {
                 />
               </h3>
             </EuiTitle>
+            <EuiScreenReaderOnly>
```

Comment:
The `EuiScreenReaderOnly` section isn't needed here as the text is the same as used in the `<h3>` element above it. You can just add `role="alert"` to the `EuiTitle` element above - `<EuiTitle size="s" role="alert">`.

Reply:
Hello @peteharverson, that's a good note, agree, thank you!
I'm updating this PR.